src/common/utils/synchronization.ts

```diff
@@ -11,7 +11,9 @@ const REDUX_ACTION_STRUCT = type({
 });
 
 const ENCRYPTED_ACTION_STRUCT = object({
-  data: string()
+  data: string(),
+  to: optional(string()),
```

Comment:
Should this be optional? What´s the behaviour when `to` is not specified?

Reply:
Then it is broadcast to all